### PR TITLE
Don't call Azure queue GetProperties API unnecessarily

### PR DESCRIPTION
**Pull request by RamCohen**
This PR reorders the logic of getting an Azure queue size in a way
that skips the call to get the queue properties (for getting the
ApproximateMessagesCount) if the queue has less than 32 messages.

Signed-off-by: Ram Cohen <ram.cohen@gmail.com>

### CODE REVIEW

**Review by JorTurFer — COMMENTED** [review]
LGTM!,
Let's wait till e2e tests finish

**Review by JorTurFer — DISMISSED** [review]
LGTM!
We will keep this PR open till we have released v2.6.1 because we'd like to release in it only bugfixes.
When we have release it, I'll ping you again for updating the changelog.

Thanks for your contribution ❤️

**Review by JorTurFer — CHANGES_REQUESTED** [review]
LGTM!
We will keep this PR open till we have released v2.6.1 because we'd like to release in it only bugfixes.
When we have release it, I'll ping you again for updating the changelog.

Thanks for your contribution ❤️

**Comment by JorTurFer** [comment]
hey @RamCohen 
Could you update the changelog.md?

**Comment by JorTurFer** [comment]
There is a conflict in the changelog.
Could you solve it? (I'll merge this once is ready)

**Review by JorTurFer — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -34,15 +34,16 @@
 
 ### Improvements
 
-- Improve e2e tests reliability ([#2580](https://github.com/kedacore/keda/issues/2580))
+- **General:** Fix generation of metric names if any of ScaledObject's triggers is unavailable ([#2592](https://github.com/kedacore/keda/issues/2592))
```
Comment (JorTurFer) [inline CHANGELOG.md:37]: This entry if from v2.6.1. remove it from here please

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,15 +34,16 @@
...
+- **Azure Queue:** Don't call Azure queue GetProperties API unnecessarily ([#2613](https://github.com/kedacore/keda/pull/2613))
 
 ### Breaking Changes
 
 - TODO ([#XXX](https://github.com/kedacore/keda/issue/XXX))
 
 ### Other
 
-- TODO ([#XXX](https://github.com/kedacore/keda/issue/XXX))
+- **General:** Improve e2e tests reliability ([#2580](https://github.com/kedacore/keda/issues/2580))
```
Comment (JorTurFer) [inline CHANGELOG.md:46]: Have you changed this from `### Improvements` to `### Other` because of any reason?
It's not a blocker, just to know it

**Review by JorTurFer — CHANGES_REQUESTED** [review]
Hi @RamCohen ,
Only small nits 😄

File: CHANGELOG.md (hunk 2 above)
Comment (RamCohen) [reply CHANGELOG.md:45]: Yes. I saw that for 2.6.1 such an item was under 'Other' (Fix failng tests...) so wanted it to be consistent

File: CHANGELOG.md (hunk 1 above)
Comment (RamCohen) [reply CHANGELOG.md]: done

**Review by JorTurFer — APPROVED** [review]
❤️ ❤️ ❤️

**Pull request merged**